presto-main/src/main/java/io/prestosql/execution/TaskInfo.java

```diff
@@ -41,6 +44,7 @@
     private final OutputBufferInfo outputBuffers;
     private final Set<PlanNodeId> noMoreSplits;
     private final TaskStats stats;
+    private final Map<String, TupleDomain<String>> dynamicTupleDomains;
```

Comment:
it doesn't need to be part of `TaskInfo` since it's already in `TaskStatus`

Reply:
Removed